app/settings/forms.py

```diff
@@ -2,9 +2,15 @@
 from wtforms import StringField, BooleanField, SubmitField, DateField, RadioField
 from wtforms.validators import ValidationError, DataRequired, Email
 from app.models import User
+from app import db
+from app.auth.forms import RegistrationForm as authenticator
+from flask_login import current_user
+import re
+
 
 
 class PreferencesForm(FlaskForm):
+    """ Form for user viewing and editting their preferences. """
```

Comment:
Can you put in the documentation what the parameter "FlaskForm" do?

Reply:
In this case its not a parameter, this is python Syntax for `PreferencesForm extends FlaskForm`.  I added documentation to reflect this.